### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aspdye**
First commit. Not finished yet. Feedback welcome :) #267 
cc @georgehrke @henni @jancborchardt

### CODE REVIEW

**Comment by aspdye** [comment]
@georgehrke Please provide me a newer screenshot of the Calendar-App and put it into the screenshot-repository ;)

File: README.md [hunk 1]
```diff
@@ -1,66 +1,22 @@
-README
-======
+# ownCloud Calendar 
 
-[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/build-status/master)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)[![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/build-status/master)
```
Comment (georgehrke) [inline README.md]: please change from calendar-rework to calendar
(here as well as the following two lines)

**Comment by aspdye** [comment]
Done @georgehrke

File: README.md (hunk 1 above)
Comment (aspdye) [inline README.md]: Done :smile:

**Comment by georgehrke** [comment]
please add a link to this FAQs :)
https://github.com/owncloud/calendar/wiki/FAQs

**Comment by georgehrke** [comment]
fixes #159

File: README.md [hunk 2]
```diff
@@ -1,68 +1,29 @@
...
+[![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 
+**A calendar app for [ownCloud](https://owncloud.org).**  
 
-This is the rework of the calendar app. <br>
-It will replace the [old calendar app](https://github.com/owncloud/calendar) with ownCloud 9.0.
+It will replace the [old calendar app](https://github.com/owncloudarchive/calendar) with ownCloud 9.0, and is only the frontend for the new CalDAV-Backend which has moved into the [ownCloud Core](https://github.com/owncloud/core). 
 
-It's __in development__, don't use it in production.
+![](https://raw.githubusercontent.com/owncloud/screenshots/master/calendar/calendar.png)
```
Comment (georgehrke) [inline README.md]: please update the image path

File: README.md [hunk 3]
```diff
@@ -1,68 +1,29 @@
...
 
-### Maintainers:
+## Maintainers:
 
  - [Georg Ehrke](https://github.com/georgehrke)
  - [Raghu Nayyar](https://github.com/raghunayyar)
  - [Bernhard Fröhler](https://github.com/codeling)
 
-### Joining Development
+If you’d like to join, just go through the [issue list](https://github.com/owncloud/calendar/issues) and fix some. :) We’re also in [#owncloud-calendar on freenode IRC](https://webchat.freenode.net/?channels=owncloud-calendar).
```
Comment (georgehrke) [inline README.md]: please add a line break after `and fix some. :)`

(Just add two spaces and hit enter)

**Comment by aspdye** [comment]
Done :wink:

**Comment by georgehrke** [comment]
Looks good to me :)
Thx a lot!!

Will squash and merge by hand

**Pull request closed without merging**